modules/gatsby/src/components/layout/table-of-contents.jsx

```diff
@@ -196,11 +197,11 @@ const SafeLink = ({
 
   return (
     <TocEntry $depth={depth} title={name} onClick={() => toggleEntry(id)}>
-      {hasChildren && <TocChevron $depth={depth} />}
+      {hasChildren && <TocChevron $depth={depth} $isTocOpen={isTocOpen} />}
       {!path || typeof path !== 'string' ? (
-        <span>{name}</span>
+        <TocHeader $depth={depth}>{name}</TocHeader>
       ) : (
-        <TocLink $depth={depth} to={path} title={name}>
+        <TocLink $depth={depth} to={path} title={name} activeStyle={activeTocLinkStyle}>
```

Comment:
i understand you try to style a link differently if it corresponds to the page displayed. 
 there's other ways we can do that w/o using activeStyle. renderRoute passes to SafeLink an "active" prop that can work, we just don't have it declared in SafeLink :|